src/elements-format.ts

```diff
@@ -207,6 +207,7 @@ export interface ElementMixin extends ElementLike {  //
    * e.g. `MyMixin`, `Polymer.PaperInputMixin`
    */
   name: string;
+  privacy: 'public'|'private'|'protected';
```

Comment:
Not in elements-format, as the old version of our json schema generator didn't work with `type` declarations it seemed. The new one does though, fixed in the next PR.